### ISSUE TITLE
Suggestions for Challenges to Solve

### ISSUE THREAD

**Issue by omniaNS** [MIT-Emerging-Talent/ET6-foundations-group-26#17]
Here are a few problem suggestions to solve. Pick one that hasn’t been taken yet, create an issue for it, assign it to yourself, and add it to the project board.

### Suggested Problems:

1. **FizzBuzz Problem**  
   print "Fizz" for multiples of 3, "Buzz" for multiples of 5, "FizzBuzz" for both, and the number itself otherwise.
   *(Assigned to: @Nada-saad635)*

2. **Palindrome Checker**  
   Check if a string is a palindrome (reads the same forwards and backwards).
   *(Assigned to: @CaesarGhazi )*

3. **Add Binary**  
   Given two binary strings a and b, return their sum as a binary string.
   *(Assigned to: @SiSaR-Pal )*

4. **Majority Element**  
   Given an array of size n, return the majority element. 
  "The majority element is the element that appears more than ⌊n / 2⌋ times."
  *(Assigned to: @AseelOmer)*

5. **Single Number**  
   Given an array of integers, every element appears twice except for one. Find that single one.

6. **Smallest Even Multiple**
  Given a positive integer, return the smallest positive integer that is a multiple of both 2 and the input integer.

7. **Convert the Temperature**
  Convert a given temperature in Celsius to Kelvin and Fahrenheit and return the result as an array.
*(Assigned to: @Elshikh-Amro)*

8. **Count Vowels**
  Given a string, count the number of vowels in it.
  *(Assigned to: @AseelOmer)*

9. **String Compression** 
  Compress a string by counting consecutive repeating characters ("aaabb" -> "a3b2").

10. **Reverse a String**
  Given a string, return the string reversed.
  *(Assigned to: @Nada-saad635)*
  
11. **Caesar Encryption** 
  Encrypt a text using the Caesar Cipher. 
  *(Assigned to: @CaesarGhazi naturally 😎)*

12. **Reverse Integer**
  Reverse the digits of a signed 32-bit integer, ensuring it stays within the 32-bit integer range. Return 0 if the reversed value exceeds the range.
  *(Assigned to: @SiSaR-Pal)*
  
 13. **Search Insert Position**
  Given a sorted array of distinct integers and a target value, return the index where the target is found or the index where it should be inserted to maintain order.
  *(Assigned to: @omniaNS)*
  
14. **Happy Number**
  determine if a number is happy.
  *(Assigned to: @omniaNS)*
---

### How to Add Suggestions:

If you have new challenges to suggest, **comment on this issue** with your idea, and I will add it to the list.

---

Happy coding! 🚀💻

**Comment by omniaNS** [comment]
@CaesarGhazi @Nada-saad635 @AseelOmer @Sufi-to @Elshikh-Amro @SiSaR-Pal @heba3

**Comment by CaesarGhazi** [comment]
Caesar Encryption
Encrypts a text using the Caesar Cipher.

i know it's a cliche but it's nice 😭😭😭

**Comment by Nada-saad635** [comment]
thank you Omnia seems nice good work ill take reverse string too

**Comment by Sufi-to** [comment]
Nice collection Omnia. We can work on other problems if we want to, right?

**Comment by omniaNS** [comment]
> Nice collection Omnia. We can work on other problems if we want to, right?

Yes absolutely, you can come up with your own problems, just make sure to add them to the board so we can keep track

**Comment by AseelOmer** [comment]
i will take the majority element
